matchbox/database/__init__.py

```diff
@@ -1,21 +1,33 @@
-import firebase_admin
-from firebase_admin import firestore
+from google.cloud import firestore
 
 from matchbox.database import error
 
+_DEFAULT_DATABASE = "(default)"
```

Comment:
I don't think that this is required because this is DEFAULT value in google.cloud library

Reply:
I think it's always better to be explicit but I don't insist on this...